Azure/communication-ui-library-android
# [Feature] screen share zoom

## Purpose

* This branch has code to scale screen share view. With scale, on double click screen is zoomed where click action is performed.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Start screen share on groups/ teams call
* Scale screen to zoom content (gesture)
* Screen should be scaled with no grey scale
* double click anywhere to zoom specific portion
